jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/AssetGroupStatsCollector.java

```diff
@@ -66,18 +67,6 @@ public List<Map<String, String>> collectAssetGroupStats() {
             errorList.add(errorMap);
             return errorList;
         }
-        Map<String, List<String>> assetGroupMap;
```

Comment:
MAIN QUESTION OF THE PR
@akashpaladin Could you please explain why exactly you got rid of this piece of code? Why it cannot be just reused to collect stats for separate datasources?

Reply:
Actually this way I was able to reduce the no .of API calls